examples/cmd/attributes.go

```diff
@@ -232,7 +256,7 @@ func addAttribute(cmd *cobra.Command) error {
 func removeAttribute(cmd *cobra.Command) error {
 	s, err := newSDK()
 	if err != nil {
-		slog.Error("could not connect", slog.String("error", err.Error()))
+		slog.Error("could not connect", "err", err)
```

Comment:
unknown which approach to do here.  I think Sean likes the original

Reply:
yeah I'll do a follow up PR at some point that makes these consistent